lib/modules/betteReddit.js

```diff
@@ -200,6 +205,9 @@ addModule('betteReddit', function(module, moduleID) {
 				default:
 					break;
 			}
+			if (module.options.truncateLongLinks.value) {
+				RESUtils.addCSS('overflow: hidden;text-overflow: ellipsis;white-space: nowrap;');
```

Comment:
@andytuba from what I understand, do you mean adding the SCSS selectors to the string that is passed in the addCSS function? i.e. `.link .title`?